packages/dam-app-base/src/Editor/SortableComponent.tsx

```diff
@@ -121,164 +128,206 @@ const AdditionalDataDisplay = ({ additionalData }: AdditionalDataDisplayProps) =
   );
 };
 
-const DragHandle = SortableHandle<DragHandleProps>(
```

Comment:
This component stayed the same, other than removing the `SortableHandle<DragHandleProps>` wrapper that was exposed by `react-sortable-hoc`.